src/main/app/claims/models/claim.ts

```diff
@@ -117,9 +117,11 @@ export class Claim {
   }
 
   get eligibleForCCJ (): boolean {
-    return !this.countyCourtJudgmentRequestedAt
-      && !this.respondedAt
-      && isPastResponseDeadline(MomentFactory.currentDateTime(), this.responseDeadline)
+    return this.isFullAdmissionPayImmediatelyPastPaymentDate()
```

Comment:
shouldn't we check for CCJRequestedAt at each case?

```
return !this.countyCourtJudgmentRequestedAt
        && (this.isFullAdmissionPayImmediatelyPastPaymentDate()
            || this.hasDefendantNotSignedSettlementAgreement()
            || (!this.respondedAt
                && isPastResponseDeadline(MomentFactory.currentDateTime(), this.responseDeadline)
                )
            )
```

Reply:
it is being checked in each case. note the use of `&& ()` so all the ors are inside the brackets and will be evaluated against the `!this.countyCourtJudgmentRequestedAt  &&`